packages/office-ui-fabric-react/src/components/Coachmark/PositioningContainer/PositioningContainer.types.ts

```diff
@@ -2,9 +2,12 @@ import { DirectionalHint } from '../../../common/DirectionalHint';
 import { IRefObject, IBaseProps, IPoint, IRectangle } from '../../../Utilities';
 import { IPositionedData } from '../../../utilities/positioning';
 
-export interface IPositioningContainer {}
+export interface IPositioningContainer { }
 
-export interface IPositioningContainerTypes extends IBaseProps<IPositioningContainer> {
+/** @deprecated */
```

Comment:
Add text after with a recommendation on the replacement

Reply:
Fixed